code/processes/gateway.q

```diff
@@ -637,4 +643,15 @@ neg[h](`.gw.asyncexecjpt;(`.q.system;"sleep 10");enlist[`servertype]!enlist`rdb`
 h(`.gw.syncexec;"`$last .z.x";enlist[`tables]!enlist enlist`logmsgXXX)
 h(`.gw.syncexec;"`$last .z.x";`tables`servertype!(enlist`data;`rdb`hdb))
 
+\d .gwreload
+
+// dictionary of handles to reload
+reloadcalls:()!();
+
+// function to add handle to reloadcalls dictionary
+po:{[h] if[.z.u in `wdb;reloadcalls[h]:0b]};
+.z.po:{[f;x] @[f;x;()];.gwreload.po x} @[value;`.z.po;{{}}];
 
+// function to remove handle from reloadcalls dictionary
+pc:{[h] reloadcalls _: h; if[(all .gwreload.reloadcalls) & count .gwreload.reloadcalls;reload[]]};
+.z.pc:{[f;x] @[f;x;()];.gwreload.pc x} @[value;`.z.pc;{{}}];
```

Comment:
I think this section won't be executed/do anything due to the `/` on line 597 - did this work when it was tested?